Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+#!groovy
+
+// Jenkinsfile for compiling, testing, and packaging the Enki libraries.
+// Requires CMake plugin from https://github.com/davidjsherman/aseba-jenkins.git in global library.
+
+pipeline {
+	agent label:'' // use any available Jenkins agent
+	stages {
+		stage('Prepare') {
+			steps {
+				sh 'mkdir -p build dist'
+				dir('enki') {
+					checkout scm
+				}
+				stash includes: 'enki/**', excludes: '.git', name: 'source'
+			}
+		}
+		stage('Compile') {
+			steps {
+				unstash 'source'
+				CMake([buildType: 'Debug',
+					   sourceDir: '$workDir/enki',
+					   buildDir: '$workDir/build/enki',
+					   installDir: '$workDir/dist',
+					   getCmakeArgs: [ '-DBUILD_SHARED_LIBS:BOOL=ON' ]
```

Comment:
We could make that the default in CMakeLists,txt if we think it should be, and so keep the Jenkins file small. Or we could build with static libraries, if we think that because Enki has not a stable API, which is the case. What do you think?